lib/build-cloud.rb

```diff
@@ -220,16 +223,31 @@ def recursive_interpolate_config(h)
         when String
 
             while h =~ /%\{(.+?)\}/
```

Comment:
What are the ^ and $ anchors in this context? It's possible there may be multiple instances of values to be interpolated, anywhere in the string.